apps/backoffice-v2/src/domains/alerts/hooks/mutations/useAlertsDecisionByIdsMutation/useAlertsDecisionByIdsMutation.tsx

```diff
@@ -0,0 +1,49 @@
+import { useMutation, useQueryClient } from '@tanstack/react-query';
+import toast from 'react-hot-toast';
+import { t } from 'i18next';
+import { AlertState, TAlertState, updateAlertsDecisionByIds } from '@/domains/alerts/fetchers';
+
+const getToastAction = (decision: TAlertState) => {
+  if (decision === AlertState.REJECTED) {
+    return 'reject_alerts' as const;
+  }
+
+  if (decision === AlertState.NOT_SUSPICIOUS) {
+    return 'not_suspicious_alerts' as const;
```

Comment:
extract 
not_suspicious_alerts , reject_alerts, revert_decision_alerts to a const or enum

Reply:
Thoughts on importing the translations JSON?